drivers/flash/flash_gd32.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022 BrainCo Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_FLASH_FLASH_GD32_H_
+#define ZEPHYR_DRIVERS_FLASH_FLASH_GD32_H_
+
+#include <errno.h>
```

Comment:
doesn't seem we use errno here, but we need `<stdint.h>`

Reply:
done.